geodiff/src/geodiff.h

```diff
@@ -597,14 +597,10 @@ GEODIFF_EXPORT void GEODIFF_V_destroy(
  * returns geometry as WKB. More details at https://www.geopackage.org/spec/#gpb_format
  * gpkgWkb is a geometry encoded according to GeoPackage spec
  * length is a length of the passed geometry buffer
- * returns buffer containing WKB encoded geometry. Caller should free that buffer by calling GEODIFF_free
+ * wkb is ooutput wkb
```

Comment:
typo: "ooutput"; also write that wkb must be already allocated to size `length (- 1?)` (? or what should be the size ?)